ansible/example_site_secrets.yml

```diff
@@ -115,7 +119,7 @@ project_app_root: '{{project_user_home }}/{{ project_name }}'
 # Username for chroot SFTP user
 sftp_user: "upload"
 # Username that should be the primary owner of the chroot SFTP directory area
-sftp_chroot_owner: "{{ project_user }}"
+sftp_chroot_owner: "{{ project_run_user }}"
```

Comment:
I think I'm going to start treating changes to example_site_secrets.yml as breaking changes by default, and make changes to the file require a major version release as well. I don't want that to prevent changes to the file, but I want a better version number attached to this repo.